pkg/apis/eventing/v1alpha1/trigger_types.go

```diff
@@ -28,6 +28,8 @@ import (
 )
 
 const (
+	// In Trigger annotation, we will use key "knative.dev/dependency" to mark the sources that the Trigger depends on
```

Comment:
Field comments should start with the field name, e.g.

`// DependencyAnnotation is the annotation key used to mark the sources that the Trigger depends on.`